tests/rule_based_profiler/domain_builder/test_categorical_column_domain_builder.py

```diff
@@ -88,34 +87,46 @@ def test_single_batch_one_cardinality(alice_columnar_table_single_batch_context)
     assert domains == alice_all_column_domains
 
 
-@mock.patch("great_expectations.data_context.data_context.DataContext")
-@mock.patch("great_expectations.core.batch.BatchRequest")
 def test_unsupported_cardinality_limit(
-    mock_data_context: mock.MagicMock,
-    mock_batch_request: mock.MagicMock,
+    alice_columnar_table_single_batch_context,
 ):
```

Comment:
Tests like these may be better suited to live under `tests/integration`, but I dont think we are currently making this separation really strictly.